incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -174,27 +198,13 @@ func (r *ObjectReconciler) getSourceInst(ctx context.Context, log logr.Logger, i
 	src.SetGroupVersionKind(inst.GroupVersionKind())
 	err := r.Get(ctx, srcNm, src)
 	if err != nil {
-		if errors.IsNotFound(err) {
-			log.Info("The source object no longer exists", "inheritedFrom", srcNS)
-			return true, nil, nil
-		} else {
+		if !errors.IsNotFound(err) {
 			log.Info("Couldn't read", "source", srcNm)
```

Comment:
Use `log.Error(err, "Couldn't read", "source", srcNm)`. Right now, the error is lost completely.

Reply:
I will keep this log as info since I changed the func back to return error.